h2o-py/h2o/h2o.py

```diff
@@ -320,7 +323,7 @@ def upload_file(path, destination_frame=None, header=0, sep=None, col_names=None
     return H2OFrame()._upload_parse(path, destination_frame, header, sep, col_names, col_types, na_strings)
 
 
-def import_file(path=None, destination_frame=None, parse=True, header=0, sep=None, col_names=None, col_types=None,
+def import_file(path=None, pattern="", destination_frame=None, parse=True, header=0, sep=None, col_names=None, col_types=None,
```

Comment:
Hmm, I don't think we had the discussion about this API at all...
In my opinion there is no need for a new parameter at all: if the user passes a path that looks like a glob (e.g. `"data/*.csv"`) then do pattern-matching; otherwise look for the file directly. Alternatively, you can check if the `path` parameter supplied is of a "regular expression" class, in which case treat it as such.